README.md

```diff
@@ -0,0 +1,133 @@
+
+- Create class Person.
+- Create class Student.
+- Create class Teacher.
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 School Library <a name="about-project"></a>
+
+**School Library** In this project you will learn how to use a module inside your class.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md">Ruby instalation instructions</a></li>
+  </ul>
+</details>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+School Library.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Ruby 3.0.1
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+You don't need to install this project.
+
+### Usage
+
+run ruby school-library.rb
+
+### Run tests
+
+No testing implemented yet
+
+### Deployment
+
+No deployment implemented yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Marco Baquero**
+
+- GitHub: [@mrcbq](https://github.com/mrcbq)
+- Twitter: [@mrcbq](https://twitter.com/mrcbq)
+- LinkedIn: [mrcbq](https://www.linkedin.com/in/mrcbq/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+More data
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+**I would like to thank to Microverse for giving this inspiring project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.

```

Comment:
- [ ] MIT License is broken, let's ensure to fix it to make our ReadMe stand out even better!

Reply:
It's Done!